samples/rag-aisearch/csharp-ooproc/FilePrompt.cs

```diff
@@ -29,38 +32,23 @@ public class SemanticSearchRequest
     // REVIEW: There are several assumptions about how the Embeddings binding and the SemanticSearch bindings
     //         work together. We should consider creating a higher-level of abstraction for this.
     [Function("IngestFile")]
-    public static async Task<SemanticSearchOutputResponse> IngestFile(
+    public static async Task<HttpResponseData> IngestFile(
         [HttpTrigger(AuthorizationLevel.Function, "post")] HttpRequestData req,
-        [EmbeddingsInput("{FilePath}", InputType.FilePath, Model = "%EMBEDDING_MODEL_DEPLOYMENT_NAME%")] EmbeddingsContext embeddings)
+        [EmbeddingsStoreInput("{URL}", InputType.URL, "{Title}", "AISearchEndpoint", "openai-index", Model = "%EMBEDDING_MODEL_DEPLOYMENT_NAME%")] EmbeddingsContext embeddings)
     {
         using StreamReader reader = new(req.Body);
         string request = await reader.ReadToEndAsync();
 
         EmbeddingsRequest? requestBody = JsonSerializer.Deserialize<EmbeddingsRequest>(request);
 
-        if (requestBody == null)
+        if (requestBody == null || requestBody.URL == null || requestBody.Title == null)
         {
-            throw new ArgumentException("Invalid request body. Make sure that you pass in {\"filePath\": value } as the request body.");
+            throw new ArgumentException("Invalid request body. Make sure that you pass in {\"URL\": value , \"Title\": value} as the request body.");
         }
 
-        string title = Path.GetFileNameWithoutExtension(requestBody.FilePath);
-
         HttpResponseData response = req.CreateResponse(HttpStatusCode.OK);
-        await response.WriteAsJsonAsync(new { status = "success", title, chunks = embeddings.Count });
-
-        return new SemanticSearchOutputResponse
-        {
-            HttpResponse = response,
-            SearchableDocument = new SearchableDocument(title, embeddings)
-        };
-    }
-
-    public class SemanticSearchOutputResponse
```

Comment:
just curious why are we removing the output binding now?

Reply:
discussed offline